templates/privacy/public.tpl

```diff
@@ -0,0 +1,64 @@
+<p>Welcome to the Account Creation tool (ACC). We recognize that you are a volunteer or wish to become a volunteer
+    working to contribute towards the world's largest free online encyclopedia. As such, we recognize that there may be
+    some information you'd rather keep private. We value that privacy and wish to assure you that we have taken steps to
+    ensure that by requesting an account here, you are not at risk of exposing your identity on the internet. At the
+    same time, however, in order to properly process your account request, we do need to collect certain information
+    that will allow us to distinguish you from others editing from the same or a nearby location.</p>
+
+<h3>Wikimedia Labs Disclaimer</h3>
+
+<p>By using this project, you agree that any private information you give to this project may be made publicly available
+    and not be treated as confidential.</p>
+<p>By using this project, you agree that the volunteer administrators of this project will have access to any data you
+    submit. This can include your IP address, your username/password combination for accounts created in Labs services,
+    and any other information that you send. The volunteer administrators of this project are bound by the Wikimedia
+    Labs Terms of Use, and are not allowed to share this information or use it in any non-approved way.</p>
+<p>Since access to this information is fundamental to the operation of Wikimedia Labs, these terms regarding use of your
+    data expressly override the Wikimedia Foundation's Privacy Policy as it relates to the use and access of your
+    personal information.</p>
+
+<h3>What data do we collect, and why?</h3>
+<p>This system records your IP Address and useragent data. This information may be processed overseas and your data will
```

Comment:
May want to explain in simple terms what useragent data is (e.g., "information about what software you're using to access this website, and the operating system your computer runs").